python313Packages.iamdata: 0.1.202510201 -> 0.1.202510211

fabaff opened the pull request:
Diff: https://github.com/cloud-copilot/iam-data-python/compare/v0.1.202510201...v0.1.202510211

Changelog: https://github.com/cloud-copilot/iam-data-python/releases/tag/v0.1.202510211


<!--
^ Please summarise the changes you have done and explain why they are necessary here ^

For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform:
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- Tested, as applicable:
  - [ ] [NixOS tests] in [nixos/tests].
  - [ ] [Package tests] at `passthru.tests`.
  - [ ] Tests in [lib/tests] or [pkgs/test] for functions and "core" functionality.
- [ ] Ran `nixpkgs-review` on this PR. See [nixpkgs-review usage].
- [ ] Tested basic functionality of all binary files, usually in `./result/bin/`.
- Nixpkgs Release Notes
  - [ ] Package update: when the change is major or breaking.
- NixOS Release Notes
  - [ ] Module addition: when adding a new NixOS module.
  - [ ] Module update: when the change is significant.
- [ ] Fits [CONTRIBUTING.md], [pkgs/README.md], [maintainers/README.md] and other READMEs.

[NixOS tests]: https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests
[Package tests]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests
[nixpkgs-review usage]: https://github.com/Mic92/nixpkgs-review#usage

[CONTRIBUTING.md]: https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md
[lib/tests]: https://github.com/NixOS/nixpkgs/blob/master/lib/tests
[maintainers/README.md]: https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md
[nixos/tests]: https://github.com/NixOS/nixpkgs/blob/master/nixos/tests
[pkgs/README.md]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md
[pkgs/test]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/test

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

fabaff committed:
python313Packages.iamdata: 0.1.202510201 -> 0.1.202510211

Diff: https://github.com/cloud-copilot/iam-data-python/compare/v0.1.202510201...v0.1.202510211

Changelog: https://github.com/cloud-copilot/iam-data-python/releases/tag/v0.1.202510211

jopejoe1 reviewed (approved):
`prowler` fails to build on darwin, but that seems like a failing reverse dependencies, so this is good with me.

## `nixpkgs-review` result for [#454261](https://github.com/NixOS/nixpkgs/pull/454261)

Generated using [`nixpkgs-review-gha`](https://github.com/Defelo/nixpkgs-review-gha)

Command: `nixpkgs-review pr 454261`
Commit: [`a9e76a7564113c95a860ada1560b7f88ec45fbcc`](https://github.com/NixOS/nixpkgs/commit/a9e76a7564113c95a860ada1560b7f88ec45fbcc) ([subsequent changes](https://github.com/NixOS/nixpkgs/compare/a9e76a7564113c95a860ada1560b7f88ec45fbcc..pull/454261/head))
Merge: [`f70101b455a94759bd19f411ac3f04cfbc05e37d`](https://github.com/NixOS/nixpkgs/commit/f70101b455a94759bd19f411ac3f04cfbc05e37d)

Logs: https://github.com/jopejoe1/nixpkgs-review-gha/actions/runs/18713889643


---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 10 packages built:</summary>
  <ul>
    <li>prowler</li>
    <li>prowler.dist</li>
    <li>python312Packages.iamdata</li>
    <li>python312Packages.iamdata.dist</li>
    <li>python312Packages.py-iam-expand</li>
    <li>python312Packages.py-iam-expand.dist</li>
    <li>python313Packages.iamdata</li>
    <li>python313Packages.iamdata.dist</li>
    <li>python313Packages.py-iam-expand</li>
    <li>python313Packages.py-iam-expand.dist</li>
  </ul>
</details>

---
### `aarch64-linux`
<details>
  <summary>:white_check_mark: 10 packages built:</summary>
  <ul>
    <li>prowler</li>
    <li>prowler.dist</li>
    <li>python312Packages.iamdata</li>
    <li>python312Packages.iamdata.dist</li>
    <li>python312Packages.py-iam-expand</li>
    <li>python312Packages.py-iam-expand.dist</li>
    <li>python313Packages.iamdata</li>
    <li>python313Packages.iamdata.dist</li>
    <li>python313Packages.py-iam-expand</li>
    <li>python313Packages.py-iam-expand.dist</li>
  </ul>
</details>

---
### `x86_64-darwin` (sandbox = true)
<details>
  <summary>:x: 2 packages failed to build:</summary>
  <ul>
    <li>prowler</li>
    <li>prowler.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 8 packages built:</summary>
  <ul>
    <li>python312Packages.iamdata</li>
    <li>python312Packages.iamdata.dist</li>
    <li>python312Packages.py-iam-expand</li>
    <li>python312Packages.py-iam-expand.dist</li>
    <li>python313Packages.iamdata</li>
    <li>python313Packages.iamdata.dist</li>
    <li>python313Packages.py-iam-expand</li>
    <li>python313Packages.py-iam-expand.dist</li>
  </ul>
</details>

---
### `aarch64-darwin` (sandbox = true)
<details>
  <summary>:x: 2 packages failed to build:</summary>
  <ul>
    <li>prowler</li>
    <li>prowler.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 8 packages built:</summary>
  <ul>
    <li>python312Packages.iamdata</li>
    <li>python312Packages.iamdata.dist</li>
    <li>python312Packages.py-iam-expand</li>
    <li>python312Packages.py-iam-expand.dist</li>
    <li>python313Packages.iamdata</li>
    <li>python313Packages.iamdata.dist</li>
    <li>python313Packages.py-iam-expand</li>
    <li>python313Packages.py-iam-expand.dist</li>
  </ul>
</details>

fabaff committed:
python313Packages.iamdata: 0.1.202510211 -> 0.1.202510221

Diff: https://github.com/cloud-copilot/iam-data-python/compare/v0.1.202510211...v0.1.202510221

Changelog: https://github.com/cloud-copilot/iam-data-python/releases/tag/v0.1.202510221
